chapters/annotations.tex

```diff
@@ -1064,24 +1060,26 @@ \section{Annotations for the Graphical User Interface}\doublelabel{annotations-f
  annotation(defaultComponentName = "name")
 \end{lstlisting}
 
-When creating a component of the given class, the recommended component
-name is name.
+When creating a component of the given class, the recommended component name is \emph{name}.
```

Comment:
Should perhaps later use a better meta-syntactic name than "name".

Reply:
I wouldn't be against it.  Pick one!